### PR TITLE
LEON has vegan and vegetarian options, no longer operating in the US

### PR DESCRIPTION
**Pull request by codeinabox**
LEON has vegan and vegetarian options on its menu https://leon.co/menu/

It has also pulled out of the US market https://www.thecaterer.com/news/leon-pulls-america-states-market-pandemic-toll-2021

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
Hello, welcome to contribute to NSI

And I have a question, is this restaurant mainly offer vegetarian/vegan food ?

If not, maybe you can refer to the quarrel in #4855 

Besides, is necessary to add both vrgan and vegetarian 2 tag?

**Comment by codeinabox** [comment]
> And I have a question, is this restaurant mainly offer vegetarian/vegan food ?

I was recently at this restaurant and I would say almost half the menu is vegan/.

That's a good question about both tags, I can update this to add just diet:vegan

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -3129,7 +3129,9 @@
         "brand:wikipedia": "en:Leon Restaurants",
         "cuisine": "burger;sandwich",
         "name": "LEON",
-        "takeaway": "yes"
+        "takeaway": "yes",
+        "diet:vegetarian": "yes",
```
Comment (LaoshuBaby) [inline data/brands/amenity/fast_food.json:3133]:
```suggestion
```

**Review by LaoshuBaby — APPROVED** [review]

**Comment by bhousel** [comment]
I removed the `diet:vegan` tags in 5574e610d

Until #4855 is resolved, please don't add `diet:vegan=yes` or `diet:vegetarian=yes` to restaurants that happen to offer a few vegan or vegetarian choices.

**Comment by bhousel** [comment]
Its ok!  I still think `diet:vegan=only` or `diet:vegetarian=only` tags have value for signaling that a brand only serves this kind of food.  

My worry is that - almost all restaurants offer _some_ vegan or vegetarian option, and if we start adding those `=yes` tag here, they will get pushed a million times into the OSM database, and people will complain.

**Pull request merged**